dask_sql/physical/rex/core/call.py

```diff
@@ -765,6 +765,18 @@ def date_part(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class BetweenOperation(Operation):
```

Comment:
Did calcite automatically convert a between operation into a combination of other ops?

Reply:
Yes, previously we were directly receiving the `<>` operator from Calcite but DataFusion just returns a `Between` variant. so that is why this is needed now.